<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FullCalculator</class>
 <widget class="QMainWindow" name="FullCalculator">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>350</width>
    <height>261</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>FullCalculator</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_5">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_5">
      <item>
       <widget class="QLineEdit" name="Monitor"/>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout">
              <item>
               <widget class="QPushButton" name="B_One">
                <property name="text">
                 <string>1</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="B_Two">
                <property name="text">
                 <string>2</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="B_Three">
                <property name="text">
                 <string>3</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_2">
              <item>
               <widget class="QPushButton" name="B_Four">
                <property name="text">
                 <string>4</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="B_Five">
                <property name="text">
                 <string>5</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="B_Six">
                <property name="text">
                 <string>6</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_3">
              <item>
               <widget class="QPushButton" name="B_Seven">
                <property name="text">
                 <string>7</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="B_Eight">
                <property name="text">
                 <string>8</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="B_Nine">
                <property name="text">
                 <string>9</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QPushButton" name="B_Zero">
            <property name="text">
             <string>0</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
             <widget class="QPushButton" name="B_Plus">
              <property name="text">
               <string>+</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="B_Minus">
              <property name="baseSize">
               <size>
                <width>2</width>
                <height>2</height>
               </size>
              </property>
              <property name="text">
               <string>-</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="B_Multiplication">
              <property name="text">
               <string>*</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="B_Division">
              <property name="text">
               <string>/</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QPushButton" name="B_Result">
            <property name="text">
             <string>=</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="B_Clean">
        <property name="text">
         <string>Clean</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>350</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
